conf/nginx.conf

```diff
@@ -47,7 +47,7 @@ location ^~ __PATH__/ {
   #rewrite ^/.well-known/host-meta.json __PATH__/public.php?service=host-meta-json last;
 
   location __PATH__/ {
-    rewrite ^ __PATH__/index.php$request_uri;
+    rewrite ^ __PATH__/index.php;
```

Comment:
Thanks for the patch :+1: 

Do you know if that specific change will work without upgrading the php version ?

Reply:
i just apply this change on my production server which runs in stable version of yunohost
For information here is the version of php that I use with this modification:

PHP 7.0.33-0+deb9u5 (cli) (built: Sep 18 2019 09:55:34) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.0.33-0+deb9u5, Copyright (c) 1999-2017, by Zend Technologies